external-crates/move/crates/move-package/src/source_package/manifest_parser.rs

```diff
@@ -339,9 +339,10 @@ pub fn parse_dependency(dep_name: &str, mut tval: TV) -> Result<PM::Dependency>
             bail!("Resolver name is not a string")
         };
 
-        // Not relevant except for the external resolver, but remove it to mark it as a
+        // Fields not relevant except for the external resolver, but remove it to mark it as a
         // recognised part of the manifest.
         let _ = table.remove("packages");
+        let _ = table.remove("network");
```

Comment:
I'd prefer not to carve out `network` as another special case. Originally, the idea was that anything under `packages` was a free-for-all which is why we were removing that, but maybe we can just say that if there is a `resolver` field, anything else is a free-for-all?

Reply:
> Originally, the idea was that anything under packages was a free-for-all

I get it, but the name isn't great for the free-for-all value, because I feel like it's natural to designate `packages` with... `packages`. It's like:

```
[dependencies.mvr]
resolver = "mvr"

[dependencies.mvr.packages]
network = "testnet"

[dependencies.mvr.packages.packages_but_actually]
Foo = "..."
Bar = "..."
```

So we can put `network` inside `packages` I get it but then what to use for the list of packages, maybe `ids`? Anyway, bit awkward.

Not a hill I am willing to die on, so will just revert this and we'll do something else.

Reply:
I understand, but what about the suggestion at the end?

> but maybe we can just say that if there is a resolver field, anything else is a free-for-all?

I.e. if you see a `resolver` field, then it means that that dependency could contain any other fields (so we are also dropping the requirement that it includes a field called `packages`).

Reply:
as we chatted about, will make this change in a separate PR!